packages/allow-scripts/src/types/common.ts

```diff
@@ -1,15 +1,29 @@
+import type { JsonObject, PackageJson } from 'type-fest';
+
 declare global {
+  export interface PkgLavamoatConfig {
+    ['allowBins']?: JsonObject;
```

Comment:
`PackageJson` is using `JsonObject` so using the same pattern here makes the unified type `LavamoatPackageJson` more consistently typed.